__mock__/mockOrchestratorStatusDal.ts

```diff
@@ -21,6 +22,9 @@ class MockOrchstratorStatusDal {
             message
         });
     }
+    async getStatusObject(uid: string, workflow: string) {
```

Comment:
This function should return a pre-staged response

Reply:
I've modified this to be a jest function, with the idea that the default behavior is not having a status object to parse, I've added an example of providing data in the OrchestratorUtils spec file on the test labeled "already marked as complete"